java/java-analysis-impl/src/com/intellij/codeInspection/dataFlow/java/inst/MethodCallInstruction.java

```diff
@@ -379,10 +390,20 @@ else if (myContext instanceof PsiMethodReferenceExpression) {
     DfaValue precalculated = getPrecalculatedReturnValue();
     PsiType type = getResultType();
 
+    if (InheritanceUtil.isInheritor(type, JAVA_UTIL_STREAM_BASE_STREAM)) {
+      DfType customObject = customObject(TypeConstraints.TOP, DfaNullability.fromNullability(myReturnNullability),
```

Comment:
Unfortunately, `CONSUMED_STREAM.asDfType` created only NOT_NULL type, and I can't find how I can change it with `meet` calls. I want to support also Nullable type, because I want to support this case:
```java

  public static void test11E() {
    Stream<Integer> integerStream = test11Em();
    Stream<Integer> integerStream1 = integerStream.<warning descr="Method invocation 'filter' may produce 'NullPointerException'">filter</warning>(t -> true);
    Stream<Integer> integerStream2 = <warning descr="Stream has already been linked or consumed">integerStream</warning>.filter(t -> true);
  }

  @Nullable
  public static Stream<Integer> test11Em() {
    return null;
  }
```

Reply:
I see. Well, there's DfReferenceType::dropNullability but this indeed complicates things. Ok, let's keep it as is for now, thanks.